src/itemRadio/index.tsx

```diff
@@ -2,17 +2,17 @@ import React, { Component, Fragment } from 'react'
 import cc from 'classcat'
 
 import Item from '_utils/item'
-import Checkbox from '_utils/checkbox'
+import Radio from '_utils/radio'
 import { TextDisplayType } from 'text'
 
 import style from './style'
 
-export enum ItemDeclaredChoiceStatus {
+export enum ItemRadioStatus {
   DEFAULT = 'default',
```

Comment:
Same thing as the CheckboxItem component: it may be better to use ItemStatus (or at least a put these in common)